stages/org.osbuild.grub2

```diff
@@ -157,8 +173,8 @@ def main(tree, options):
 
     with open(grubenv, "w") as env:
         env.write("# GRUB Environment Block\n"
-                  f"GRUB2_ROOT_FS_UUID={root_fs_uuid}\n"
-                  f"GRUB2_BOOT_FS_UUID={root_fs_uuid}\n"
+                  f"GRUB2_ROOT_FS_UUID={grub_fs_uuid}\n"
+                  f"GRUB2_BOOT_FS_UUID={grub_fs_uuid}\n"
```

Comment:
I lost track now, why do we need both again?

Reply:
The [`blscfg`](https://github.com/rhboot/grub2/blob/d26095b02308647059363ced7545867f24698bf0/grub-core/commands/blscfg.c#L1019) command in is current form needs it:

```c
#ifdef GRUB_MACHINE_EMU
    devid = "host";
#elif defined(GRUB_MACHINE_EFI)
    devid = grub_env_get ("root");
#else
    devid = grub_env_get ("boot");
#endif
```
@martinezjavier made a patch to get rid of it though, so we can drop it in the future.

Reply:
As we introduced these variables, would it make sense to already unify them now and make the change instead to where the variables are used and use `"search --no-floppy --fs-uuid --set=boot ${GRUB2_ROOT_FS_UUID}\n"`? Or is this way somehow more semantically correct?

Reply:
I think we could and probably should do just `set boot=$root` to avoid doing the extra search.